doc/reference/tooling/tt_cli/configuration.rst

```diff
@@ -69,6 +71,13 @@ app section
     Default: ``var/lib``.
 *   ``vinyl_dir`` -- the directory where vinyl files or subdirectories are stored.
     Default: ``var/lib``.
+
+    .. note::
+
+        In all directories specified in ``*_dir`` parameters, ``tt`` creates a
+        directory for each application and instance directories inside it.
+        Names is these directories match the names of applications and instances.
```

Comment:
Did you mean "Names of these directories..."?

Reply:
Sure, thanks

Reply:
Looks like, not pushed yet.